lib/auth/signup/sign_up_page.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:chat_app_front/auth/signup/sign_up_strings.dart';
+import 'package:chat_app_front/auth/user_repository.dart';
+import 'package:chat_app_front/auth/user_repository_impl.dart';
+import 'package:flutter/material.dart';
+
+import 'sign_up_form.dart';
+import 'sign_up_user.dart';
+
+class SignUpPage extends StatefulWidget {
+  final UserRepository userRepository = UserRepositoryImpl();
+
+  @override
+  _SignUpPageState createState() => _SignUpPageState(userRepository);
+}
+
+class _SignUpPageState extends State<SignUpPage> {
+  final UserRepository userRepository;
+
+  _SignUpPageState(this.userRepository);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text(SignUpStrings.appBarText),),
+      body: Container(
+        child: ListView(
```

Comment:
Wydaje mi się, że możesz bezpośrednio jako child podać SignUpForm

Reply:
Zrobiłem w ten sposób żeby potem dodać inne widgety np. ten tekst already member please log in, który nie jest częścią formularza rejestracyjnego